sale_order_general_discount_payment_term/models/account_payment_term.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class AccountPaymentTerm(models.Model):
+    _inherit = "account.payment.term"
+
+    sale_discount = fields.Float(
+        store=True,
+        readonly=False,
+    )
```

Comment:
No point restating obvious default IMO. (This is a new field, not an override, correct?)

Instead, I think it'd be good to add the same modifiers that the corresponding field has in `sale_order_general_discount`

```py
    sale_discount = fields.Float(
        digits=dp.get_precision('Discount'),
        string='Discount (%)',
        company_dependent=True,
        track_visibility="onchange",
    )
```